packages/headless/src/controllers/search-box/headless-search-box.test.ts

```diff
@@ -239,6 +240,14 @@ describe('headless searchBox', () => {
       expect(engine.actions).toContainEqual(deselectAllBreadcrumbs());
     });
 
+    it('allows autoSelection', () => {
+      searchBox.submit();
+
+      expect(engine.actions).toContainEqual(
+        updateFacetAutoSelection({allow: true})
```

Comment:
It would be good to assert the order of the actions, since someone inverting the order would introduce a regression.

e.g. index of `updateFacetAutoSelection` > index of `deselectAllBreadcrumbs` in `engine.actions`